index.js

```diff
@@ -458,15 +459,57 @@ class ClientServiceBase {
   /**
    * Execute a registered contract
    * @param {string} contractId
-   * @param {Object} argument
-   * @param {Object} [functionArgument=undefined]
+   * @param {Object} contractArgument
+   * @param {Object} [functionArgument=null]
+   */
+  async executeContract(contractId, contractArgument, functionArgument) {
```

Comment:
Can we add something like `@Deprecated`?

Reply:
Sorry, my bad!
Thank you for the reminder.
I added it in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/83/commits/aa4a4d095ebc9a8cd435f84606ce86896543297e

Please take a look again!